PackageNamespacesExample/Lib2/Lib2.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="A" Version="5.0.0" />
+    <PackageReference Include="B" Version="3.0.0" />
+    <PackageReference Include="C" Version="3.0.0" />

```

Comment:
I think we should add some real packages here so that when customers download this sample and run restore, they can see namespaces in action.

Reply:
Those are real packages from the `nuget-build` source. 

Majority of the packages on that source are on nuget.org as well. I'm sure there's some better options, but they'll all be obvious test names.

Reply:
Sorry I didn't know about that those packages are real. I am good now.

Reply:
What is `nuget-build` source here?

Reply:
Also `https://www.nuget.org/packages/A` is unlisted.

Reply:
> What is `nuget-build` source here?

The change is in the PR. There's a nuget.config there.

> Also `https://www.nuget.org/packages/A` is unlisted.

Unlistedness does not affect restore. The package `A` is getting download from a different source anyways.